changelog.md

```diff
@@ -4,239 +4,244 @@
 
```

Comment:
Are you using some formatter here?
I think it is better to keep using * for list

Reply:
It seems so, yes, I wasn't aware of this md formatter. Reverting this file.